src/main/java/billennium/tests/service/quiz/QuizServiceImpl.java

```diff
@@ -47,7 +47,8 @@ public QuizModel findQuiz(UUID userId) {
 
     @Override
     @Transactional
-    public void updateQuizStatus(Long quizId, UUID userId) {
+    public void updateQuizStatus(Long quizId, UUID userId) { //TODO tutaj też lecą optionale bez sprawdzenia i
```

Comment:
Tak bo póki co to jest raczkowanie projektu i miałem zawsze tylko jeden obiekt który wiedziałem że jest

Reply:
i właśnie takie przypadki powinny być badane testami